projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/src/components/EmailForm/style.scss

```diff
@@ -46,6 +46,10 @@
 					border: solid 1px #e65054;
 				}
 
+				&:focus-visible {
+					border: 1px solid var(--verbum-focus-color) !important;
```

Comment:
Maybe we can change this to `outline` since will work better across themes

Reply:
Updated